src/gitea/cli/main.py

```
@@ -101,9 +101,11 @@ def main(
 def register_commands() -> None:
     """Register CLI commands."""
     from gitea.cli.config.main import config_app  # noqa: PLC0415
+    from gitea.cli.issue.main import issue_app  # noqa: PLC0415
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)